sample/src/main/java/com/davidecirillo/multichoicesample/sampleToolbar/MySampleToolbarAdapter.java

```diff
@@ -89,4 +89,11 @@ public void onClick(View v) {
     public int getItemCount() {
         return mList.size();
     }
+
+    public void swap(ArrayList<String> data) {
+        mList.clear();
+        mList.addAll(data);
+
+        refreshDataSet();
+    }
 }
```

Comment:
This name is not self explaining what this method is doing, no documentation, what is the logic behind it?